### DISCUSSION TITLE
How to install it ?

### DISCUSSION THREAD

**Discussion by adomej99** [nlohmann/json#2432] [Q&A] [answered]
How can I install this package on windows using visual studio?

**Comment by nlohmann** [comment, marked as answer]
It's a single header - you do not need to install it. But if you want to, you can install it just like any other CMake project.

**Reply by nlohmann** [reply]
Do you need further assistance?

**Reply by MusicNearMe** [reply]
Hey quick question here, btw thanks for the intricate tool @nlohmann. 
I am also aiming to convert JSON data coming to over to C++, and I'm a noob at coding, but a have an engineering background. 

After adding the header file into an UNREAL project in VS, I see the VS compiler says assert is undefined. A comment mentions defining NDEBUG at compile time.... how would that get properly defined? Is it simply adding the line #define NDEBUG in the code?

**Reply by nlohmann** [reply]
Please add a new issue for this and add as many details as possible.
